### PR TITLE
Format AttestationCleint.java

### PR DESCRIPTION
**Pull request by rbehjati**
More formatting and linting to make g3 happy.

### CODE REVIEW

**Review by ipetr0v — APPROVED** [review]

File: oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java [hunk 1]
```diff
@@ -1,15 +1,12 @@
 package com.google.oak.functions.client;
 
 import com.google.common.base.VerifyException;
-import com.google.oak.functions.client.AeadEncryptor;
-import com.google.oak.functions.client.KeyNegotiator;
```
Comment (ipetr0v) [inline oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java:5]: Why are these lines removed? Can Java see classes from the same package by default?

**Review by ipetr0v — APPROVED** [review]

File: oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java (hunk 1 above)
Comment (rbehjati) [reply oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java:5]: Yes. So these imports are redundant.
